src/main/java/net/minecraftforge/fml/common/registry/IForgeRegistry.java

```diff
@@ -40,6 +40,8 @@
 {
     public Class<V> getRegistrySuperType();
 
+    ResourceLocation getRegistryName();
+
```

Comment:
Rather not add this to the user facing API, the entire point is to hide things like this.